containers/unit_tests/TestUnorderedMap.hpp

```diff
@@ -560,6 +560,49 @@ TEST(TEST_CATEGORY, UnorderedMap_constructor_view_alloc) {
   ASSERT_TRUE(map.is_allocated());
 }
 
+//////////////////////////Tests for UnorderedMap with View as value_type
+
+/**
+ * @test This test ensures that an @ref UnorderedMap with View as value_type can
+ * be built with an execution space instance (using @ref view_alloc).
+ */
+TEST(TEST_CATEGORY, UnorderedMap_View_as_value) {
+  using map_type =
+      Kokkos::UnorderedMap<int, Kokkos::View<size_t *, TEST_EXECSPACE>,
+                           Kokkos::HostSpace>;
```

Comment:
It will make it more readable when you insert values.
```suggestion
  using value_type = Kokkos::View<size_t *, TEST_EXECSPACE>;
  using map_type = Kokkos::UnorderedMap<int, value_type, Kokkos::HostSpace>;
```